hvplot/plotting/core.py

```diff
@@ -329,26 +414,36 @@ def scatter(self, x=None, y=None, **kwds):
             plotted, then only the first color from the color list will be used.
         **kwds : optional
             Additional keywords arguments are documented in `hvplot.help('scatter')`.
-        
+
+        Returns
+        -------
+        A Holoviews object. You can `print` the object to study its composition and run `hv.help` on
+        the object to learn more about its parameters and options.
+
         Example
         -------
 
         >>> import hvplot.pandas
         >>> from bokeh.sampledata.iris import flowers as df
         >>> df.hvplot.scatter(
-        ...     x='sepal_length', y='sepal_width', by='species', 
+        ...     x='sepal_length', y='sepal_width', by='species',
         ...     legend='top', height=400, width=400
         ... )
 
         The points will be grouped and color coded `by` the categorical values in the 'species'
         column.
-        
-        Returns
-        -------
-        A Holoviews object. You can `print` the object to study its composition and run `hv.help` on
-        the object to learn more about its parameters and options.
+
+        References
+        ----------
+
+        - Bokeh: http://docs.bokeh.org/en/latest/docs/user_guide/plotting.html#scatter-markers
+        - HoloViews: http://holoviews.org/reference/elements/matplotlib/Scatter.html
+        - Pandas: https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.plot.scatter.html
+        - Plotly: https://plotly.com/python/line-and-scatter/
+        - Matplotlib:  https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.scatter.html
+        - Wiki: https://en.wikipedia.org/wiki/Scatter_plot
```

Comment:
Do you know if there's a way to automatically check that these links don't 404?

Reply:
Yes. Python and pytest. I will add a test.

Reply:
Please note that for me these links are the only way to understand hvPlot. They are a life saver. The current documentation is so sparse that I have no idea on how it should work, what options are available or how things can be combined. The only way to understand hvPlot is by getting the perspective from the underlying frameworks.

But finding the alternative information takes a long time because things are not called the same across. So you have to spend a considerable amount of googling.